### PR TITLE
Update page building dependencies and instructions

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

File: doc/pages/README.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+bundle exec jekyll build
+```
+
+5. Commit and push the updated version
+
+```
+cd public
+git add -A
+git commit -m Update
+git push origin master
```
Comment (iNecas) [inline doc/pages/README.md]: I personally prefer the convention that [Hub](https://github.com/github/hub) uses ('origin' for the shared repo, and `$MYUSERNAME` for the forks), just a suggestion: not insisting

**Comment by iNecas** [comment]
Shouoln't this be opened directly against the master branch? If I understand that correctly, we don't even need the `pages` branch anymore.

**Comment by adamruzicka** [comment]
Waiting for https://github.com/Dynflow/dynflow/pull/274

**Comment by iNecas** [comment]
There are conflicts, please rebase

**Comment by iNecas** [comment]
You probably meant waiting for #275 - this is merged now.

**Comment by adamruzicka** [comment]
Off by one error :) It should be good to go now

**Comment by iNecas** [comment]
Thanks @adamruzicka

**Pull request merged**